common/src/main/java/io/druid/indexer/Jobby.java

```diff
@@ -19,9 +19,24 @@
 
 package io.druid.indexer;
 
+import javax.annotation.Nullable;
+import java.util.Map;
+
 /**
  */
 public interface Jobby
 {
   boolean run();
+
+  @Nullable
```

Comment:
Would you please add a javadoc describing when the return value can be null?

Reply:
Added javadoc